pkgs/tools/misc/diffoscope/default.nix

```diff
@@ -1,25 +1,24 @@
-{ lib, stdenv, fetchgit, fetchpatch, python3, docutils
+{ lib, stdenv, fetchgit, fetchpatch, python3Packages, docutils
 , acl, binutils, bzip2, cbfstool, cdrkit, colord, cpio, diffutils, e2fsprogs, file, fpc, gettext, ghc
 , gnupg1, gzip, jdk, libcaca, mono, pdftk, poppler_utils, sng, sqlite, squashfsTools, unzip, vim, xz
```

Comment:
I think you meant to replace `vim` -> `xxd` (otherwise the reference to `xxd` below doesn't work).